app/api/routes.py

```
@@ -296,48 +285,55 @@ def ocr_and_save(
+    gcs_path: str = Query(..., description="GCS 내부 경로"),
+):
+    try:
+        # GCS에서 실제 바이트 다운로드
+        file_bytes = download_from_gcs(gcs_path)
+
+        content_type = "image/jpeg"
+        if gcs_path.lower().endswith(".png"):
+            content_type = "image/png"
+
+        return Response(content=file_bytes, media_type=content_type)
+    except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)